k8s/crds/kops.k8s.io_instancegroups.yaml

```diff
@@ -97,6 +97,10 @@ spec:
                 description: DetailedInstanceMonitoring defines if detailed-monitoring
                   is enabled (AWS only)
                 type: boolean
+              ec2InstanceConnect:
+                description: EC2InstanceConnect installs the ec2-instance-connect
```

Comment:
We should note that this is AWS-only, either by putting this stuff in a namespace or just saying so in the description.